Conventional.Samples.Tests/Legacy/Conventions.cs

```diff
@@ -10,16 +10,22 @@ public class Conventions
         [Test]
         public void MustNotUseDateTimeDirectly()
         {
-            typeof (DbContext).Assembly.GetExportedTypes()
-                .MustConformTo(Convention.MustNotResolveCurrentTimeViaDateTime);
+            typeof(DbContext).Assembly.GetExportedTypes()
+                .MustConformTo(Convention.MustNotResolveCurrentTimeViaDateTime)
+                .WithFailureAssertion(Assert.Fail);
+                
         }
 
         readonly DateTime _doomsday = new DateTime(2015, 12, 2);
 
         [Test]
         public void MustNotUseDateTimeDirectly_Alternative()
         {
-            typeof (DbContext).Assembly.GetExportedTypes()
+            // note: doomsday tests require warning and failure assertions to be set up globally
```

Comment:
This isn't ideal, in that we set up some static properties, the alternative would be a note pointing to test setup, but that also changes the WithFailureAssertion calls